wiki/christmas-trees/design_flow_process.md

```diff
@@ -5,10 +5,12 @@
 2. Attach one or more screenshots of designs (as needed) to the story and uses @symbol address to notify for review.
 3. If necessary a small adhoc meeting to demo the mockups should be held.
 4. If needed, usability testing is done on the selected mockup.
-5. TTS reviews mockups and checks the "Pre approved" task which implies that the story can be deployed to staging.
+5. Throughout the sprint, TTS reviews mockups and checks the "Pre approved" task which implies that the story can be deployed to staging.
```

Comment:
@MelissaBraxton mind making sure this reflects our process?

Reply:
Yep, LGTM!